src/service/packet_router.proto

```diff
@@ -71,6 +71,7 @@ message packet_router_packet_down_v1 {
   bytes payload = 1;
   window_v1 rx1 = 2;
   window_v1 rx2 = 3;
+  bytes gateway = 4;
```

Comment:
Why add this overhead to every packet headed to the gateway?

Reply:
No, this is an experiment to handle data from Router to HPR with less streams.

Reply:
Gotcha but it’s also the proto message that the gateway receives right? If so you may want to put the downlink packet in an envelope with the gateway in the outer envelope

Reply:
We could leave it empty when it goes to gateway

Reply:
Seems like that breaks some system boundaries in favor of some optimization. Let’s chat this through before nailing it down